### PR TITLE
handled error, completed test

### PR DESCRIPTION
**Pull request by offlineAndrew**
1. created endpoint
2. created a test
3. handled the error

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require ('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+const { response } = require('../app');
```
Comment (aloush-dev) [inline __tests__/index.test.js:6]: is this something which has auto imported?

File: __tests__/index.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+test('GET: 200 sends and array of topics', () => {
+    return request(app)
+    .get('/api/topics').expect(200)
+    .then((response) => {
+        expect(response.body.topics.length).toBe(3);
```
Comment (aloush-dev) [inline __tests__/index.test.js:17]: Great job checking the length before forEach'ing through!

File: app.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const {
+getTopics
+} = require('./controllers/topics-controllers');
+
+
+app.use(express.json());
```
Comment (aloush-dev) [inline app.js:8]: Is this needed yet as we are not dealing with anything from a request?

**Review by aloush-dev — APPROVED** [review]
Lovely.
Looks great!
For going forward start looking into some jest matchers which will help you in further tasks when testing. ObjectContaining and ToMatchObject are two good ones for these tests.

File: __tests__/index.test.js (hunk 1 above)
Comment (offlineAndrew) [reply __tests__/index.test.js:6]: hi Ali,| not sure what actually are you asking for. I checked examples from previous katas ('can you handle this' and 'michs treasures')

File: app.js (hunk 3 above)
Comment (offlineAndrew) [reply app.js:8]: yep, sorry, not needed so far

**Pull request merged**